### ISSUE TITLE
fine-tune detection  low  hmean, bad results?

### ISSUE THREAD

**Issue by chunniunai220ml** [PaddlePaddle/PaddleOCR#7028]
在用自己的数据，finetune文本检测模型的时候，P,R一直很低，像请教原因。

采用官网提供的configs/det/det_res18_db_v2.0.yml及其对应train model，标注数据格式如下

<img width="568" alt="image" src="https://user-images.githubusercontent.com/17668109/181510342-1c9950fa-fdfa-413d-bec4-743afdadf289.png">

训练日志可以看出，PR一直不增长：

<img width="1158" alt="image" src="https://user-images.githubusercontent.com/17668109/181510618-e123237f-98cb-41ce-b246-d27aff28c4f4.png">

还有这个optimizer，里面的baselr 和lr怎么理解？
<img width="204" alt="image" src="https://user-images.githubusercontent.com/17668109/181511263-3f1c4005-e19e-4e88-b245-1280bd02d14d.png">

训练的识别模型，configs/rec/PP-OCRv3/ch_PP-OCRv3_rec_distillation.yml (14.8M的模型)，这个acc
正常吗？  我在benchmark里面没有看到精度指标对比

<img width="1314" alt="image" src="https://user-images.githubusercontent.com/17668109/181511628-f5bf7ad3-740c-4d03-a8bd-292dfc32bfdc.png">

**Comment by LDOUBLEV** [comment]
More information is needed.
Which yml file are you using?
Which dataset are you using to finetune?
What does a bad case look like?

**Comment by chunniunai220ml** [comment]
> More information is needed.
> Which yml file are you using? I 
> Which dataset are you using to finetune? about 3K train images
> What does a bad case look like? wrong detection ,and. loss many dew-boxes

**Comment by LDOUBLEV** [comment]
<img width="342" alt="image" src="https://user-images.githubusercontent.com/26592129/182136667-608270d9-374f-48ee-be89-9bc4b2d0769d.png">
这个部分的标注有点问题，中间的空格间隔太大了，很难拟合为一个一个框，训练的时候每一行更容易被区分为三个框
